design-system/materials/custom-properties.css

```diff
@@ -6,44 +6,73 @@
 */
 
 :root {
-  --color-primary: #00b39e;
-  --color-primary-20: hsl(172.9608938547486, 100%, 20%);
-  --color-primary-25: hsl(172.9608938547486, 100%, 25%);
-  --color-primary-30: hsl(172.9608938547486, 100%, 30%);
-  --color-primary-40: hsl(172.9608938547486, 100%, 40%);
-  --color-primary-85: hsl(172.9608938547486, 100%, 85%);
-  --color-primary-95: hsl(172.9608938547486, 100%, 95%);
+  --color-primary: hsl(175, 55%, 45%);
+  --color-primary-20: hsl(175, 55%, 20%);
+  --color-primary-25: hsl(175, 55%, 25%);
+  --color-primary-30: hsl(175, 55%, 33%);
```

Comment:
I can see this discrepancy also in `color-info-40`, `color-warning-40` and `color-error-40`.

Personally, I don't like this too much since here we're using a number sequence based on the opacity and it would be a little more difficult to reason about if, in the future, we need the 30% again since that token would need to be lower than 30 (maybe `color-token-29`).

On the other hand, I understand we want to change our current token so everywhere is used, the value is updated.
This would make sense for semantic tokens, which I think these ones are not.

For instance, if we were dealing with simple color design tokens (`primary`, `accent`, `warning`) it would be right to change those, but as we're using different levels per color where the levels are bound to opacity in this case, it feels like mixing concepts.

It would make more sense to me to create a new token and change consumers to use it, but this would require more work.

Anyway, I'd like to read more opinions from other @commercetools/shield-team-fe team members.